src/utils/entity-state-utils.ts

```diff
@@ -64,6 +67,12 @@ export async function deleteEntityState(hubChannel: HubChannel, world: HubsWorld
   const payload: DeleteEntityStatePayload = {
     nid: APP.getString(Networked.id[eid])! as NetworkID
   };
+  const {
+    initialData: { fileId }
+  } = createMessageDatas.get(eid)!;
+  if (fileId) {
+    payload.file_id = fileId;
+  }
```

Comment:
The backend "delete_entity_state" behavior is similar to the previous "unpin" behavior in that if it receives a `file_id` it will run `OwnedFile.set_inactive`. If it doesn't receive a `file_id` it just goes straight to performing the unpinning and broadcasting "entity_state_deleted". If "delete_entity_state" is ever called for an OwnedFile without a file_id then it would be inaccessible and stored forever. This feels tenuous as it puts a lot of faith in createMessageDatas on the client. This is related to the other concern you mentioned but probably outside the scope of this PR.